Dockerfile

```diff
@@ -58,8 +70,13 @@ LABEL \
   io.k8s.display-name="${CLI_NAME}" \
   io.openshift.tags="conforma ec opa cosign sigstore"
 
-# Install tools we want to use in the Tekton task
-RUN microdnf upgrade --assumeyes --nodocs --setopt=keepcache=0 --refresh && microdnf -y --nodocs --setopt=keepcache=0 install git-core jq
+# Copy in the packages from the 'packages' stage.
+# These are the binaries needed at runtime.
+COPY --from=packages /usr/bin/jq /usr/bin/gzip /usr/bin/
```

Comment:
Consider [this](https://github.com/enterprise-contract/ec-cli/pull/955#issuecomment-1688787997) approach to retain the rpm db record to make security folks happy.